README.md

```diff
@@ -100,6 +100,7 @@ model.fit(x, y) # Keras model.
 - `dilations`: List. A dilation list. Example is: [1, 2, 4, 8, 16, 32, 64].
 - `nb_stacks`: Integer. The number of stacks of residual blocks to use.
 - `activation`: String. The activations to use in the residual blocks (norm_relu, wavenet, relu...).
+- `padding`: String. The padding to use in the convolutions. 'causal' for a causal network (as in the original implementation) and 'same' for a non-causal network.
```

Comment:
In the signature, add padding='causal'

tcn.TCN(nb_filters=64, kernel_size=2, nb_stacks=1, dilations=None, activation='norm_relu', use_skip_connections=True, dropout_rate=0.0, return_sequences=True, name='tcn')

To

tcn.TCN(nb_filters=64, kernel_size=2, padding='causal', nb_stacks=1, dilations=None, activation='norm_relu', use_skip_connections=True, dropout_rate=0.0, return_sequences=True, name='tcn')